nemo/collections/asr/models/transformer_bpe_models.py

```diff
@@ -636,5 +744,25 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
             'pin_memory': True,
         }
 
+        # TODO: remove this lhotse hardcoding later (@kpuvvada)
+        # currently only works for non-tarred
+        lhotse_config = {
+            'is_tarred': False,
+            'batch_size': 1,
+            'force_strip_pnc': False,
+            'token_sequence_format': "canary",
+            'use_lhotse': True,
+            'lhotse': {
```

Comment:
update this config namespace to reflect the changes in lhotse PR when merged to main